README.md

```diff
@@ -4,6 +4,8 @@ This repo contains work-in-progress documentation for .NET. To contribute, see t
 
 We welcome contributions to help us improve and complete the .NET docs. We anticipate that [Xamarin](http://developer.xamarin.com/api/root/classlib/), [Mono](http://docs.go-mono.com/?link=root%3a%2fclasslib) and [Unity](http://docs.unity3d.com/Manual/index.html) will also use this documentation.
 
+This repository contains the conceptual documentation for .NET. There are separate repositories for the [code samples and snippets](https://github.com/dotnet/samples) and [API reference](https://github.com/dotnet/dotnet-api-docs). Issues and tasks for all three of those repositories are tracked here.
```

Comment:
Section **Samples Build Status** at the end of this file should be removed or updated to report status of another repo. Or that is in the scope of another PR?